src/android/LocalFilesystem.java

```diff
@@ -471,4 +471,53 @@ public boolean canRemoveFileAtLocalURL(LocalFilesystemURL inputURL) {
 		File file = new File(path);
 		return file.exists();
 	}
+
```

Comment:
I've left this commented out for now, but the code moved to cordova-android. This does need to be tested still but the API has moved in the file already

Reply:
Please ensure we add a test case that would have run this code

Reply:
Ok will add a subtask for adding to our code base - i don't want to add here